input/resources/au-core-actor-requester.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ActorDefinition xmlns="http://hl7.org/fhir">
+  <id value="au-core-actor-requester"/>
+  <url value="http://hl7.org.au/fhir/core/ActorDefinition/au-core-actor-requester" />
+  <name value="AUCoreRequester"/> 
+  <title value="AU Core Requester"/> 
+  <status value="draft"/>
+  <description value="The AU Core Requester is a system that responds to the data access request submitted by requesters, providing responses to requests for core digital health and administrative information."/> 
+  <type value="system"/> 
+  <documentation value="An AU Core Client, also known as AU Core Requester:&#xa;&#xa;
+- **SHALL** accept resources containing any valid value for *Must Support* elements without error&#xa;&#xa;
+- **SHALL** accept resources containing *Must Support* elements with [Missing data](general-requirements.html#missing-data) or [Suppressed Data](general-requirements.html#suppressed-data) without error&#xa;&#xa;
+How the system processes the resource depends on local requirements that could align with obligation terms such [reject invalid](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-reject-invalid), [correctly handle](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-handle), [persist](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-persist), [display](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-display), or [ignore](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-ignore).&#xa;&#xa;
```

Comment:
A word is missing; change to: "How the system processes the resource depends on local requirements that could align with obligation terms such _+as+_ [reject invalid](https://hl7.org/fhir/extensions/CodeSystem-obligation.html#obligation-reject-invalid), ..."

Reply:
fixed